code/game/objects/items/devices/radio/radio.dm

```diff
@@ -207,6 +208,8 @@
 				channels[chan_name] &= ~FREQ_LISTENING
 			else
 				channels[chan_name] |= FREQ_LISTENING
+	else if (href_list["bold"])
+		use_command = text2num(href_list["bold"])

```

Comment:
since it's just a toggle I could always remove the href_list value entirely and just make it a use_command != use_command.

Only did it this way to keep consistency with other bits of the file. Will change.